### ISSUE TITLE
updateByQuery removed from 5.0 branch

### ISSUE THREAD

**Issue by ericofusco** [elastic/elasticsearch-php#503]
### Summary of problem or feature request

On ElasticSearch-PHP 2.2.3 there is the method updateByQuery, although it was removed from 5.0.0 branch.

update_by_query still exists on ElasticSearch 5.0.

https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-update-by-query.html

### Code snippet of problem
```php
$es->updateByQuery($params);
```

It gives the following error:
```
Exception 'Elasticsearch\Common\Exceptions\BadMethodCallException' with message 'Namespace [updateByQuery] not found' in /var/www/erico/vendor/elasticsearch/elasticsearch/src/Elasticsearch/Client.php:1288
```

### System details

- Operating System: Ubuntu 14.04
- PHP Version: 7.0
- ES-PHP client version: 5.0.0
- Elasticsearch version: 5.0.0

**Comment by polyfractal** [comment]
Heh, yeah, the saga of Update/Delete by query :)

It was removed from core, then reimplemented.  I'll add the endpoint back ASAP... thanks for the poke!

**Comment by ericofusco** [comment]
Thanks a lot @polyfractal. Looking forward to get this back. :)

**Comment by erikvanderschaaf** [comment]
Any estimate on this one?

**Comment by sandyandi** [comment]
@polyfractal please review my PR. I merged master to my branch so `DeleteByQuery` gets fixed in branch `5.0` as well then I put back `UpdateByQuery`.

**Comment by polyfractal** [comment]
Merged!
